packages/client/utils/relay/LocalPokerHandler.ts

```diff
@@ -1,23 +1,45 @@
-
 import {Handler} from 'relay-runtime/lib/store/RelayStoreTypes'
+import {TaskStatusEnum, TaskServiceEnum} from '~/types/graphql'
+import upperFirst from '../upperFirst'
 import createProxyRecord from './createProxyRecord'
 
-const LocalPokerHandler: Handler = {
-  update(store, payload) {
-    const meetingId = payload.dataID
-    const meeting = store.get(meetingId)!
-    const queryId = `jiraSearchQuery:${meetingId}`
-    const existingQuery = store.get(queryId)
+const lookup = {
+  [TaskServiceEnum.jira]: {
+    meetingPropertyName: 'jiraSearchQuery',
+    defaultQuery: {
+      queryString: '',
+      projectKeyFilters: [],
+      isJQL: false
+    }
+  },
+  [TaskServiceEnum.PARABOL]: {
+    meetingPropertyName: 'parabolSearchQuery',
+    defaultQuery: {
+      queryString: '',
+      statusFilters: [TaskStatusEnum.active]
+    }
+  }
+}
+
+const initializeDefaultSearchQueries = (store, payload): void => {
+  const meetingId = payload.dataID
+  const meeting = store.get(meetingId)!
+
+  for (const [, data] of Object.entries(lookup)) {
+    const {meetingPropertyName, defaultQuery} = data
+    const existingQuery = meeting.getLinkedRecord(meetingPropertyName)
     if (!existingQuery) {
-      const jiraSearchQuery = createProxyRecord(store, 'JiraSearchQuery', {
-        id: queryId,
-        queryString: '',
-        projectKeyFilters: [],
-        isJQL: false
-      })
-      meeting.setLinkedRecord(jiraSearchQuery, 'jiraSearchQuery')
+      const newQuery = createProxyRecord(
+        store, upperFirst(meetingPropertyName), defaultQuery)
```

Comment:
this makes the assumption that the type name of the search query will be the same as its property name upon meeting.